src/App.tsx

```diff
@@ -143,20 +147,50 @@ const Test = connect(
         onLearn: (item: model.Learnable) => dispatch(actions.learn(item)),
         onReview: (id: model.LearnableId, correct: boolean) =>
             dispatch(actions.review(id, correct)),
-        onWander: () => dispatch(actions.wander()),
+        onWander: () => {
+            dispatch(actions.wander());
+            dispatch(actions.modifyResource(resources.STAMINA, resources.WANDER_STA));
+        },
+        onMeditate: () => {
+            dispatch(actions.modifyResource(resources.STAMINA, resources.MEDITATE_STA));
+        },
         handleEventEffect: (effect: event.Effect) => dispatch(effect.toAction())
     })
 )(TestComponent as React.ComponentType<TestProps>);
 
-class App extends React.Component {
-  render() {
-    return (
-      <div className="App">
-          <h1>Michael Mauer Simulator 2017</h1>
-          <Test/>
-      </div>
-    );
-  }
+interface AppProps {
+    dispatch: Dispatch<actions.Action>;
 }
 
+class BaseApp extends React.Component<AppProps> {
```

Comment:
As I stated before, `index.tsx` actually has the store itself, and you can dispatch actions against that object. You don't need to do all this ceremony in React. Redux is its own library, and the store created in `index.tsx` has a dispatch method.

Reply:
addressed in 56aa5f9